tests/test_login.py

```diff
@@ -19,10 +19,12 @@
 
 import hydra_base
 import hydra_base.exceptions
+import hydra_server
```

Comment:
We don't import hydra server into hydra base. It introduces a circular dependency which we don't want.
I've simply make these checks for 'HydraErrors'. While these are less specific, the only alternative is to move the authorization and authentication errors to hydra base, which is a separate project